packages/components/workbench/Workbench.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Workbench'
+type: 'component'
+status: 'stable'
+slug: /components/workbench/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/workbench'
+storybook: 'https://f36-storybook.contentful.com/?path=/docs/components-workbench--basic'
+typescript: ./Workbench.tsx,./WorkbenchHeader/WorkbenchHeader.tsx,./WorkbenchContent/WorkbenchContent.tsx,./WorkbenchSidebar/WorkbenchSidebar.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+An app shell that structures regions of content and interaction.
+The workbench assembles the outer app shell which defines regions to structure content and interactions.
+This component is not included in `@contentful/f36-components`
+which means that to use, you will need to import it from `@contentful/f36-workbench` directly.
+
+## Table of contents
+
+- [Header](#header)
+- [Main content](#main-content)
+- [Sidebars](#sidebars)
+- [Props of Workbench](#props-of-workbench)
+- [Props of Workbench.Header](#props-of-workbenchheader)
+- [Props of Workbench.Content](#props-of-workbenchcontent)
+- [Props of Workbench.Sidebar](#props-of-workbenchsidebar)
+
+## Header
+
+The Header provides navigational guidance and primary actions on the current view.
+
+### Content guidelines
+
+- The Header should only provide actions which are global to the current view (e.g. Button to create a user on a users list view)
+- When multiple global actions are needed a dropdown menu should be used
+- Buttons that submit a form on the current view should not be placed in the title bar but be part of that specific form.
+
+## Main content
+
+The main content area provides the main functionality of the current view.
+
+### Content guidelines
+
+Depending on the displayed information one of the 3 width configurations should be applied:
+
+- Views that just consist of forms or text should have a maximum width of 768px for enhanced readability.
+- Static table views which contents are not configurable should default to a maximum width of 1280px.
+- Rich data views like dashboards or configurable tables should use the entire available width.
+
+## Sidebars
+
+The left sidebar provides quick access to saved views and filters.
+The right sidebar provides secondary interactions depending on the current view as well as documentation spaces and entry points for extensibility.
+
+### Content guidelines
+
+- The left sidebar serves as secondary navigation and provides predefined and user-defined filters
+- It should not include any view actions like adding or removing content
+
+## Props of Workbench
+
+<Props of="Workbench" />
+
+## Props of Workbench.Header
+
+<Props of="WorkbenchHeader" />
+
+## Props of Workbench.Content
+
+<Props of="WorkbenchContent" />
```

Comment:
I don't see props table showing up in the documentation website.

Reply:
Yep, I'm also not seeing them
I thought someone would be able to tell me why

Reply:
Oh I discovered why! hahahahah

I forgot to put `./src` in the path of the TS files in the .mdx